assets/service_broker/Gemfile.lock

```diff
@@ -41,3 +41,6 @@ DEPENDENCIES
   rainbow
   rspec
   sinatra
+
+BUNDLED WITH
+   2.1.4
```

Comment:
I think it would be better to avoid locking this down if possible so that we don't need to worry about the ruby/bundler version that is used when deploying this app.